docs/sources/tempo/api_docs/metrics-summary.md

```diff
@@ -10,15 +10,16 @@ weight: 600
 # Metrics summary API
 
 {{< admonition type="warning" >}}
-The Metrics summary API is an [experimental feature](/docs/release-life-cycle) that is disabled by default. To enable it, adjust your configuration as suggested below.
+The metrics summary API is deprecated in Tempo 2.7. Features powered by the metrics summary API, like the Aggregate by table, are also deprecated in Grafana Cloud and Grafana 11.3 and later.
```

Comment:
```suggestion
The metrics summary API is deprecated as of Tempo 2.7. Features powered by the metrics summary API, like the Aggregate by table, are also deprecated in Grafana Cloud and Grafana 11.3 and later.
```

Reply:
Not sure if this is more readable? Feel free to reject.

Reply:
I like that better. Thank you for the suggestion.